sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -12346,7 +12499,7 @@
+          "calculateValue": "/**\r\n * Part-time is always considered \"independant\".\r\n * Full-time student will be considered \"independant\" if\r\n * parent information does not need to be entered after \r\n * all the questions in the personal information are answered or \r\n * if parent information needs to be entered and student\r\n * \t- has both the parents passed away or; \r\n * \t- is estranged from both of parents. \r\n * Otherwise, student is considered \"dependant\".\r\n */\r\n\r\n// If the intensity is Part-time, student is always considered 'independant'.\r\nif (data.howWillYouBeAttendingTheProgram === \"Part Time\") {\r\n  instance.setValue(\"independant\");\r\n}\r\n// If the intensity is Full-time, dependant status is calculated based on personal information answers.\r\nelse if (data.howWillYouBeAttendingTheProgram === \"Full Time\") {\r\n  const isStudentIndependant =\r\n    data.hasDependents === \"yes\" ||\r\n    [\"married\", \"other\"].includes(data.relationshipStatus) ||\r\n    data.outOfHighSchoolFor4Years === \"yes\" ||\r\n    data.fulltimelabourForce === \"yes\" ||\r\n    data.parentsDeceased === \"yes\" ||\r\n    data.estranged === \"yes\" ||\r\n    data.custodyOfChildWelfare === \"yes\";\r\n  // If the student is not independant, dependant status is calculated based on personal information answer inputs.\r\n  if (!isStudentIndependant) {\r\n    const isPersonalInfoAnswered =\r\n      !!data.hasDependents &&\r\n      !!data.relationshipStatus &&\r\n      !!data.outOfHighSchoolFor4Years &&\r\n      !!data.fulltimelabourForce &&\r\n      !!data.parentsDeceased &&\r\n      !!data.estranged &&\r\n      !!data.youthInCare &&\r\n      (data.youthInCare === \"no\" || !!data.custodyOfChildWelfare);\r\n    // If the answers to required personal information inputs are not provided, set the dependant status to null.\r\n    // Otherwise, dependant status is calculated based on personal information answers.\r\n    const calculatedStatusFromPersonalInfo = isPersonalInfoAnswered\r\n      ? \"dependant\"\r\n      : null;\r\n    instance.setValue(calculatedStatusFromPersonalInfo);\r\n  } else {\r\n    instance.setValue(\"independant\");\r\n  }\r\n}\r\n// If the intensity is not provided, set the value to null.\r\nelse {\r\n  instance.setValue(null);\r\n}\r\n ",
```

Comment:
As we have the refactored code having sufficient comments, this block of comments can be removed. 

```js
/**
 * Part-time is always considered "independant".
 * Full-time student will be considered "independant" if
 * parent information does not need to be entered after 
 * all the questions in the personal information are answered or 
 * if parent information needs to be entered and student
 * 	- has both the parents passed away or; 
 * 	- is estranged from both of parents. 
 * Otherwise, student is considered "dependant".
 */
```